### PR TITLE
update roxygen2 contents

### LINKED ISSUE #680: update roxygen2 content

### LINKED ISSUE #703: don't even mention @seealso

### LINKED ISSUE #704: mention doc inheritance more

### LINKED ISSUE #705: consider mentioning the devtag package

### LINKED ISSUE #706: check "markdown currently doesn’t work in template files"

### LINKED ISSUE #709: update exampleIf mention
> @exampleIf tag present, at the time of writing, in roxygen2 development version.

### PR DESCRIPTION
**Pull request by maelle**
Fix #680 
Fix #703 
Fix #709 
Fix #704
Fix #705
Fix #706 (template tag no longer recommended by roxygen2)

### CODE REVIEW

**Comment by maelle** [comment]
In this PR, I updated the Spanish version manually + using `babeldown::deepl_translate_markdown_string()`, not `babeldown::deepl_update()`.

**Review by mpadge — APPROVED** [review]
Another strong improvement, thank you

File: pkg_building.es.Rmd [hunk 1]
```diff
@@ -258,26 +256,18 @@ f <- function(a = TRUE) {
 ```
 
 - La documentación debe ayudar a la navegación incluyendo links entre funciones relacionadas y agrupando la documentación de funciones relacionadas en página de ayuda comunes.
-  Para esto recomendamos las etiquetas `@seealso` y `@family` (esta última crea automáticamente enlaces "See also" y [puede agrupan las funciones](https://pkgdown.r-lib.org/reference/build_reference.html) en sitios web generados con pkgdown).
+  Para esto recomendamos la etiqueta `@seealso`, que crea automáticamente enlaces "See also" y [puede agrupan las funciones](https://pkgdown.r-lib.org/reference/build_reference.html) en sitios web generados con pkgdown.
```
Comment (yabellini) [inline pkg_building.es.Rmd:259]:
```suggestion
  Para esto recomendamos la etiqueta `@seealso`, que crea automáticamente enlaces _"See also"_ (ver también, en inglés) y [puede agrupan las funciones](https://pkgdown.r-lib.org/reference/build_reference.html) en sitios web generados con pkgdown.
```

File: pkg_building.es.Rmd [hunk 2]
```diff
@@ -258,26 +256,18 @@ f <- function(a = TRUE) {
...
 
 - Puedes reutilizar partes de la documentación (por ejemplo, detalles sobre la autenticación, paquetes relacionados) en las viñetas, *README* y páginas de manual.
-  Una forma de hacerlo es [usando fragmentos de R Markdown](https://www.garrickadenbuie.com/blog/dry-vignette-and-readme/).
-  Usando "documentos hijos" con kintr puedes guardar fragmentos de código en la carpeta `man` y reutilizarlos en el *README*, las viñetas y la documentación (usando la etiqueta `@includeRmd`, [disponible desde la versión 0.7.0 de roxygen2](https://roxygen2.r-lib.org/articles/rd.html#including-external--rmd-md-files)).
-
-- Añade `#' @noRd` a las funciones internas.
-
-- Si prefieres no abarrotar el código con una documentación muy larga, puedes colocar más ejemplos fuera del *script* de R.
-  Coloca la documentación extra en archivos en la carpeta `man-roxygen` en la raiz de tu paquete e insértala en el archivo de ayuda con `@template <nombre de archivo>` debajo de la etiqueta `@example`.
+  Consulta [viñeta roxygen2 sobre la reutilización de la documentación](https://roxygen2.r-lib.org/articles/reuse.html).
```
Comment (yabellini) [inline pkg_building.es.Rmd:263]:
```suggestion
  Consulta la [viñeta de roxygen2 sobre la reutilización de documentación](https://roxygen2.r-lib.org/articles/reuse.html).
```

File: pkg_building.es.Rmd [hunk 3]
```diff
@@ -258,26 +256,18 @@ f <- function(a = TRUE) {
...
-    2. Cuando utilizas la misma documentación en muchas funciones y archivos `.R` es útil utilizar una plantilla. Esto reduce el texto duplicado y ayuda a evitar que por error se actualice la documentación de una función pero no la de otra. Utilizar un archivo de plantilla para la documentación de un parámetro es una alternativa a definir el parámetro para una función y utilizar `@inheritParams` para otras funciones que utilicen el mismo parámetro.
+- Para incluir ejemplos, puedes utilizar el clásico `@examples` (plural "examples"), pero también la etiqueta `@example <path>` (singular "example") para almacenar el código de ejemplo en un script R independiente (idealmente en `man/`), y la etiqueta `@exampleIf` para ejecutar ejemplos condicionalmente y evitar fallos de `R CMD check`. Consulta [documentación de roxygen2 sobre ejemplos](https://roxygen2.r-lib.org/articles/rd.html#examples).
```
Comment (yabellini) [inline pkg_building.es.Rmd:265]:
```suggestion
- Para incluir ejemplos, puedes utilizar el clásico `@examples` (en plural _"examples"_), pero también la etiqueta `@example <path>` (en singular _"example"_) para almacenar el código de ejemplo en un script R independiente (idealmente en `man/`), y la etiqueta `@exampleIf` para ejecutar ejemplos condicionalmente y evitar fallos de `R CMD check`. Consulta [la documentación de roxygen2 sobre ejemplos](https://roxygen2.r-lib.org/articles/rd.html#examples).
```

**Review by yabellini — COMMENTED** [review]
I did some minor changes. Thanks!

**Pull request merged**